### DISCUSSION TITLE
disdinguish batch effect from real biological difference

### DISCUSSION THREAD

**Discussion by sunshx-bioinfo** [satijalab/seurat#9292] [General]
Hi,

I have data about early embryo development and there are 5 time points in this data. In each time point, several embryos are merged together before single cell RNA sequencing. If i want to remove batch effect among my data, i just can only use time points as batch, but there are real biologocal differences among them. So for that case, do i need to try to remove batch effect and how to do that?

Does anyone can give any advice :)? Thx.

**Comment by samuel-marsh** [comment]
Hi,

Not member of dev team but hopefully can be helpful.  As this is more general analysis question I'm going to move this to discussions and hopefully others can chime in.

This is general issue in use of integration in scRNA-seq analyses.  How to manage this is going to depend on the method used for integration as well as the magnitude of any technical and biological effects in the data.  Again, most integration methods are not performing corrections on the actual gene expression but integrating the embeddings and then the original gene expression data can be used to examine differences.

Best,
Sam
